deployment/terraform/config.tf

```diff
@@ -1,6 +1,6 @@
 provider "aws" {
   region  = "${var.aws_region}"
-  version = "~> 1.56.0"
+  version = "~> 2.70.0"
```

Comment:
This resolves a race condition I encountered with the creation of resources for Step Functions: https://github.com/hashicorp/terraform-provider-aws/issues/7893.

Reply:
Good catch, glad to see an upgrade!